src/Cron.php

```diff
@@ -162,6 +162,7 @@ public static function isValid(string $expression): bool
 
     public static function fromJson(string $json): Frequency
     {
+        /** @var array<string, string> $data */
```

Comment:
Why is this needed? Is this about that json_decode may return null? Then we should handle it accordingly and not mute the error with such meaningless inline doctype.

Reply:
```php
Line   Cron.php                                                              
 ------ ---------------------------------------------------------------------- 
  167    Cannot access offset 'expression' on mixed.                           
  168    Cannot access offset 'start' on mixed.                                
  169    Cannot access offset 'start' on mixed.
...
```

Reply:
Then throw an error if the result is not an array.